src/_locales/en_US/messages.json

```diff
@@ -512,5 +512,37 @@
                 "example": "Facebook Like"
             }
         }
+    },
+    "sync_intro": {
+        "message": "Cloud syncing:<ul><li>Requires Firefox/Chrome Sync</li><li>Download combines the lists of sites where your Badger is disabled</li><li>Upload overwrites the list saved in the cloud</li></ul>",
+        "description": "A brief explanation of how syncing works. Shown above the upload/download disabled sites buttons under the Manage Data options page tab."
+    },
+    "upload": {
+        "message": "Upload",
+        "description": "Button label for Sync Upload on Manage Data options page tab."
+    },
+    "upload_whitelist": {
+        "message": "Upload disabled sites to cloud",
+        "description": "Button title for Sync Upload on Manage Data options page tab."
+    },
+    "download_whitelist": {
+        "message": "Download disabled sites from cloud",
+        "description": "Button title for Sync Download on Manage Data options page tab."
+    },
+    "download_whitelist_success": {
+        "message": "Disabled sites list downloaded from cloud and merged into local list.",
+        "description": "Message displayed to the user after successfully downloading the synced whitelist."
```

Comment:
No need to mention what is being synced inside the descriptions (we may sync more than just disabled sites in the future).

"... after successfully merging cloud data."

Reply:
done